Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Defrag - centralized API for the openSUSE Infrastructure
+# Copyright (C) 2021 openSUSE contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.	See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+#!BuildTag: defrag
+
+FROM opensuse/tumbleweed
+
+ENV ENV "True"
```

Comment:
It's the first time I see this. Why is it required?

Reply:
This just sets the environment variable `ENV` to true. We could also remove it, but this would require `ENV=True` to be in the .env file. It's just there for convenience 😉

Reply:
Okay.